superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -208,6 +211,28 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
     return data;
   };
 
+  const customGotoPage = (p: number) => {
+    SupersetClient.get({
+      endpoint: `/api/v1/dataset/?q=${rison.encode({
+        order_column: 'changed_on_delta_humanized',
+        order_direction: 'desc',
+        page: p - 1,
+        page_size: 20,
+      })}`,
+    }).then(resources => {
+      setPageIndex(p - 1);
+      setResourceCollection(resources.json.result);
+    });
+  };
+
+  const styleOverride = css`
+    .table-condensed {
+      height: 300px;
```

Comment:
Do we need to lock this to 300px tall, or is it possible to let it get taller, and not need to scroll when the viewport is tall enough?

Reply:
yes or otherwise we get that weird sticky tab for the sort tabs.